### PR TITLE
Replace grpc_health_probe with a direct gRPC call in tests

### PR DESCRIPTION
**Pull request by ricopinazo**
### What changes were proposed in this pull request?
Replace grpc_health_probe with a direct gRPC call in tests

### Why are the changes needed?
Otherwise, we need to install grpc_health_probe

### Does this PR introduce any user-facing change? If yes is this documented?
No

### How was this patch tested?
The DynamicClassLoaderTest works

### Are there any further changes required?
No

### CODE REVIEW

File: test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala [hunk 1]
```diff
@@ -85,19 +91,18 @@ class DynamicClassLoaderTest extends CatsEffectSuite {
...
+  private val defaultConf: Config = ConfigBuilder.getDefaultConfig
+
+  private val waitForRaphtoryService: IO[Unit] =
+    Health.client[IO](ChannelForAddress("localhost", defaultConf.getInt("raphtory.deploy.port"))).use { client =>
+      for {
+        serving <- client.Check(HealthCheckRequest()).map(response => response.status.isServing).handleError(_ => false)
+        _       <- if (serving) IO.unit else IO.sleep(1.seconds) *> waitForRaphtoryService
+      } yield ()
+    }
+
```
Comment (ljeub-pometry) [inline test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala:103]: Can we implement this as part of the creation for the remote context?

File: test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala [hunk 2]
```diff
@@ -145,8 +150,6 @@ class DynamicClassLoaderTest extends CatsEffectSuite {
 
   case object RemoteContext extends Context
 
-  val defaultConf: Config = ConfigBuilder.getDefaultConfig
```
Comment (ljeub-pometry) [inline test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala:148]: Do you mind updating the context creation below to use the new `RaphtoryIOContext.localIO()` and `RaphtoryIOContext.remoteIO()` methods?

**Review by ljeub-pometry — DISMISSED** [review]
We can get this merged first to remove the Go dependency. The other suggestions I made can be done as part of a new pull request and are not as urgent.

File: test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala (hunk 1 above)
Comment (ricopinazo) [reply test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala:104]: @ljeub-pometry and I just discussed this. We are going to make a proper ticket for this afterwards.

**Review by miratepuffin — APPROVED** [review]

**Pull request merged**